admin/services/data-access/sql.service.js

```diff
@@ -330,8 +330,8 @@ function addParamsToRequest (params, request) {
  * @return {Promise}
  */
 sqlService.modify = async (sql, params = []) => {
-  // logger.debug('sql.service.modify(): SQL: ' + sql)
-  // logger.debug('sql.service.modify(): Params ', R.map(R.pick(['name', 'value']), params))
+  logger.debug('sql.service.modify(): SQL: ' + sql)
```

Comment:
intentional?

Reply:
Yes. I wondered if you would notice 😄 

I would be happy to hide it behind a VERBOSITY flag if needed.

so if verbosity  = 1 : standard debug log
if verbosity = 2  : sql  statement logs.

Reply:
that would be good 💯

Reply:
The env var is now DEBUG_VERBOSITY, and accepts an int.